test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/survival/KieServerWithSmartRouterAndControllerSurvivalIntegrationTest.java

```diff
@@ -54,7 +55,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-@Category({JBPMOnly.class, OperatorNotSupported.class})
+@Category({JBPMOnly.class, OperatorNotSupported.class, MonitoringK8sFs.class})
```

Comment:
Is this correct? This integration test will be executed in Templates only and the MonitoringK8sFs is only for Operator.